dashboard/test/factories.rb

```diff
@@ -2,6 +2,11 @@
 
 FactoryGirl.allow_class_lookup = false
 FactoryGirl.define do
+  factory :course do
+    name "MyString"
+    plc_course_id 1
```

Comment:
YMMV but you may want to leave plc_course_id nil. Most courses will not have these and test objects should reflect that reality